packages-ts/integration-starkgate/test/tokenBridge.test.ts

```diff
@@ -33,6 +42,9 @@ describe('Test StarkGate token bridge + link_token.cairo', function () {
   let starknetERC20Bridge: Contract
 
   before(async () => {
+    devnetNetwork = await startNetwork()
```

Comment:
We should do this similar as the funder, have some network starter class that gets the configuration for the test env and starts the appropriate networks. If a testnet is specified we won't start anything as the starter only knows to start StarkNetDevnet and HardHat.

So try to follow the Funder example and create a `NetworkManager` that reads in config/opts (we can read config from env as with Funder), and then we just do something like `manager.start(opts)` + `manager.stop()`.